common-topics/vipps-landing-page.md

```diff
@@ -102,7 +103,7 @@ the landing page.
 
 **Please note:** When using `skipLandingPage`, the user is not sent to a URL
 after completion of the payment. The "result page" is just the confirmation in
-Vipps. The `fallback` URL sent in the API request can therefore be the
+the Vipps app. The `fallback` URL sent in the API request can therefore be the
```

Comment:
The fallBack URL is in the eCom API. Maybe we need to separate the eCom and payment flows here?

Reply:
Is the new update ok?

When using `skipLandingPage`, the user is not sent to a URL
after completion of the payment. The "result page" is just the confirmation in
the Vipps app. The required parameter `returnUrl` (called `fallback` for the eCom API)
is not used, so you can provide any URL as that value.